filc/disk.go

```diff
@@ -165,7 +167,12 @@ func setup(ctx context.Context, cfgdir string) (*Node, error) {
 		return nil, err
 	}
 
-	bsnet := bsnet.NewFromIpfsHost(h, nil)
+	frt, err := fullrt.NewFullRT(h, dht.DefaultPrefix, fullrt.DHTOption(dht.Datastore(ds), dht.BucketSize(20), dht.BootstrapPeers(dht.GetDefaultBootstrapPeerAddrInfos()...)))
```

Comment:
we might want this to spin up in client mode instead of server mode, I believe theres a dht option for it that we can pass